src/protagonist/DLCS.Core/Settings/DlcsSettings.cs

```diff
@@ -28,6 +28,11 @@ public class DlcsSettings
     /// </summary>
     public string SpaceManifestQuery { get; set; }
     
+    /// <summary>
+    /// URL format for generating manifests for single assets
+    /// </summary>
+    public string SingleAssetManifestTemplate { get; set; }

```

Comment:
This value will be unlikely to change between deployments so should have a default value set in `appSettings.json` to remove the need to set it whenever it's deployed.

Reply:
Good point, changed in [0c3a4e0](https://github.com/dlcs/protagonist/pull/644/commits/0c3a4e09c0bb34121a6616d968002d04bc5610b7)